alchemy/src/alchemy.ts

```diff
@@ -34,11 +34,7 @@ function parseCliArgs(): Partial<AlchemyOptions> {
     options.phase = "read";
   }
 
-  if (
-    args.includes("--dev") ||
-    args.includes("--watch") ||
-    process.execArgv.includes("--watch")
-  ) {
+  if (args.includes("--dev")) {
     options.dev = true;
   }
```

Comment:
@john-royal is this right?

We need `--watch` to enable watching (push esbuild changes to live Worker) and `--dev` to enable local mode.

Reply:
Would that override the dev: { remote: boolean } property on resources? Replace it entirely?

Reply:
That's a good reason to keep it as --dev instead of --local

A global flag should not take precedence over an explicit `dev` config on the resource.